packages/common/src/kms.ts

```diff
@@ -16,11 +14,16 @@ class KMS {
     };
 
     const kms = new AWS.KMS();
-    const r = await kms.encrypt(params).promise();
-    return r.CiphertextBlob.toString('base64');
+    const { CiphertextBlob } = await kms.encrypt(params).promise();
+
+    if (!CiphertextBlob) {
+      throw new Error('Encryption failed, undefined CiphertextBlob returned');
+    }
+
+    return CiphertextBlob.toString('base64');
   }
 
-  static async decrypt(text) {
+  static async decrypt(text: string) {
```

Comment:
Can this whole class not be removed based on the deprecation notice?

Reply:
We can't remove it quite yet because we're waiting for [this PR](https://github.com/nasa/cumulus/pull/1713) to be released first.

Reply:
I assume you meant to link a different PR

Reply:
Nope, that's the right one. We need to make sure that a useful error message is displayed if the provider credentials migration script has not yet been run. Once that's in place, then we can move forward with #1714, which will remove the S3 keypair and KMS stuff.